backend/substrapp/serializers/ledger/aggregatealgo/serializer.py

```diff
@@ -10,7 +10,8 @@
 
 class LedgerAggregateAlgoSerializer(serializers.Serializer):
     name = serializers.CharField(min_length=1, max_length=100)
-    permissions = PermissionsSerializer()
+    permissions = PermissionsSerializer(required=False,
+                                        default={'public': False, 'authorized_ids': True})
```

Comment:
`'authorized_ids': True` seems strange to me. I'm probably missing something but isn't it supposed to be a list?

Reply:
Same thing for every other `PermissionsSerializer` in this PR

Reply:
It is absolutely supposed to be a list, sorry.